src/Mesh/babylon.mesh.ts

```diff
@@ -281,9 +281,6 @@
                     Tags.AddTagsTo(this, Tags.GetTags(source, true));
                 }
 
-                // Parent
-                this.parent = source.parent;

```

Comment:
useless, it is set by the `Tools.DeepCopy`above :
```javascript
                 Tools.DeepCopy(source, this, ["name", "material", "skeleton", "instances", "parent", "uniqueId",
                     "source", "metadata", "hasLODLevels", "geometry", "isBlocked", "areNormalsFrozen",
                     "source", "metadata", "hasLODLevels", "geometry", "isBlocked", "areNormalsFrozen",
                     "onBeforeDrawObservable", "onBeforeRenderObservable", "onAfterRenderObservable", "onBeforeDraw"
                     "onBeforeDrawObservable", "onBeforeRenderObservable", "onAfterRenderObservable", "onBeforeDraw"
                 ],
]
````

Reply:
source is in the avoid list

Reply:
will remove, not related to the PR anyway